apps/wallet/src/ui/app/components/accounts/ProtectAccountForm.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm, type SubmitHandler } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import * as Yup from 'yup';
+import { CheckboxField } from '../../shared/forms/CheckboxField';
+import { Form } from '../../shared/forms/Form';
+import { TextField } from '../../shared/forms/TextField';
+import ExternalLink from '../external-link';
+import { Button } from '_app/shared/ButtonUI';
+import { ToS_LINK } from '_src/shared/constants';
+
+const formSchema = Yup.object({
+	password: Yup.string().required('Required'),
```

Comment:
Sorry I missed this, can we use zod for new things instead?

Reply:
sure thing!